docs/development/database-migrations.rst

```diff
@@ -9,7 +9,8 @@ removing tables). To autogenerate migrations::
 Verify your migration was generated by looking at the output from the command
 above:
 
-    ``Generating /opt/warehouse/src/warehouse/migrations/versions/390811c1cdbe_.py ... done``
+    ``Generating /opt/warehouse/src/warehouse/migrations/versions/390811c1c\
```

Comment:
It doesn't look like this line break is being properly rendered. I'm not an RST expert though, so I'm not sure how to resolve the rendered `\ `

Reply:
Apparently this renders correctly when viewed _not on github_ ✨